src/Pmi/Controller/DashboardController.php

```diff
@@ -881,7 +881,7 @@ public function metricsLoadEHRAction(Application $app, Request $request)
      * Get Metrics 2 Object
      *
      * Main method for retrieving near-real-time metrics from API;
-     * Stores result in memcache with 15-minute expiration
+     * Caches result with 15-minute expiration
```

Comment:
Noting that the cache expiration is actually handled in `RdrMetrics` instead of here. Likely an old comment before a refactoring.